graphql/distro_resolver.go

```diff
@@ -42,6 +58,55 @@ func (r *distroResolver) ProviderSettingsList(ctx context.Context, obj *model.AP
 	return settings, nil
 }
 
+// Version is the resolver for the version field.
+func (r *finderSettingsResolver) Version(ctx context.Context, obj *model.APIFinderSettings) (FinderVersion, error) {
+	if obj == nil {
+		return FinderVersionLegacy, InternalServerError.Send(ctx, "distro undefined when attempting to resolve finder version")
+	}
+
+	switch utility.FromStringPtr(obj.Version) {
+	case evergreen.FinderVersionLegacy:
+		return FinderVersionLegacy, nil
+	case evergreen.FinderVersionParallel:
+		return FinderVersionParallel, nil
+	case evergreen.FinderVersionPipeline:
+		return FinderVersionPipeline, nil
+	case evergreen.FinderVersionAlternate:
+		return FinderVersionAlternate, nil
+	default:
+		return FinderVersionLegacy, InternalServerError.Send(ctx, fmt.Sprintf("finder version '%s' is invalid", utility.FromStringPtr(obj.Version)))
+	}
+}
+
+// Version is the resolver for the version field.
+func (r *plannerSettingsResolver) Version(ctx context.Context, obj *model.APIPlannerSettings) (PlannerVersion, error) {
+	if obj == nil {
+		return PlannerVersionTunable, InternalServerError.Send(ctx, "distro undefined when attempting to resolve planner version")
```

Comment:
if these nil cases are considered an error, can we just return the empty string? Returning a value implies that we're doing something with it (not sure if that's the case).